server/pages/index.js

```diff
@@ -253,7 +253,7 @@ function setUpLoggedInRoute( req, res, next ) {
 		start = new Date().getTime();
 
 		debug( 'Issuing API call to fetch user object' );
-		user( req.get( 'Cookie' ), function( error, data ) {
+		user( `wordpress_logged_in=${ req.cookies.wordpress_logged_in }`, function( error, data ) {
```

Comment:
Double checked, it should work 👌

Reply:
That part worries me the most, we should drop that commit before merging and create a separate PR